helm-chart/eoapi/test-gcp-unittest-values.yaml

```diff
@@ -1,8 +1,28 @@
 # used in GH Actions `.github/workflows/helm-tests.yml.jobs.integration-tests`
-testing: true
+<<<<<<< HEAD
```

Comment:
Looks like a broken merge conflict in this file

Reply:
doh